### PR TITLE
Improved command instructions formatting

### PR DESCRIPTION
**Pull request by Mackenzie-OO7**
## Description

This PR updates command instructions for better readability by renaming headings and putting blocks for each command under their respective steps. These changes help users follow instructions easily, understand the purpose of each step and execute commands more efficiently.

## What type of PR is this

documentation

### CODE REVIEW

File: docs/develop/getting-started/quick_start.md [hunk 1]
```diff
@@ -25,17 +25,27 @@ The easiest way to install WasmEdge is to run the following command. You should
...
+ `curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash`
+```
 
 
 ## Run a standalone Wasm app
 
 The Hello world example is a standalone Rust application that can be executed by the [WasmEdge CLI](../build-and-run/cli). Its source code and build instructions are available [here](https://github.com/second-state/rust-examples/tree/main/hello).
 
-[Download the hello.wasm file here](/files/hello.wasm)
+Download the hello.wasm file [here](/files/hello.wasm), or run the following command:
```
Comment (kokamkarsahil) [inline docs/develop/getting-started/quick_start.md:39]: This are hashed files which are bundled by webpack the original files exist at the following path: https://wasmedge.org/docs/files/hello.wasm

image

File: docs/develop/getting-started/quick_start.md [hunk 2]
```diff
@@ -25,17 +25,27 @@ The easiest way to install WasmEdge is to run the following command. You should
...
+
+```bash
+wget https://wasmedge.org/docs/assets/files/hello-ac4b04cab55321c2ccfef5cf435b6213.wasm
```
Comment (kokamkarsahil) [inline docs/develop/getting-started/quick_start.md:42]: This are hashed files which are bundled by webpack the original files exist at the following path: https://wasmedge.org/docs/files/hello.wasm

image

File: docs/develop/build-and-run/docker_wasm.md [hunk 3]
```diff
@@ -5,12 +5,12 @@ sidebar_position: 4
 # Docker + Wasm
 
 The Docker Desktop distributes with the WasmEdge Runtime embedded. That allows developers to build, share and run very lightweight containers (i.e., a `scratch` empty container with only the `.wasm` file without any Linux OS libraries or files) through Docker tools.
-Those "Wasm containers" are fully OCI-compliant and hence can be managed by Docker Hub. They are cross-platform and runs on any OS / CPU Docker supports (the OS and CPU platform is `wasi/wasm`).
-But most importantly, they are 1/10 of the size of a comparable Linux container and start up in 1/10 of the time as the Wasm containers do not need to bundle and start Linux libraries and services.
+Those "Wasm containers" are fully OCI-compliant and hence can be managed by Docker Hub. They are cross-platform and run on any OS / CPU Docker supports (the OS and CPU platform is `wasi/wasm`).
```
Comment (adithyaakrishna) [inline docs/develop/build-and-run/docker_wasm.md:8]:
```suggestion
Those "Wasm containers" are fully OCI-compliant and hence can be managed by Docker Hub. They are cross-platform and can run on any OS / CPU Docker supports (the OS and CPU platform is `wasi/wasm`).
```

File: docs/develop/getting-started/quick_start.md (hunk 2 above)
Comment (Mackenzie-OO7) [reply docs/develop/getting-started/quick_start.md]: Thank you for the review! I've made the suggested corrections.

File: docs/develop/build-and-run/docker_wasm.md (hunk 3 above)
Comment (Mackenzie-OO7) [reply docs/develop/build-and-run/docker_wasm.md]: Thank you!

File: docs/develop/getting-started/quick_start.md [hunk 4]
```diff
@@ -25,17 +25,21 @@ The easiest way to install WasmEdge is to run the following command. You should
...
-## Run a standalone Wasm app
+## How to run a standalone Wasm app
 
 The Hello world example is a standalone Rust application that can be executed by the [WasmEdge CLI](../build-and-run/cli). Its source code and build instructions are available [here](https://github.com/second-state/rust-examples/tree/main/hello).
 
-[Download the hello.wasm file here](/files/hello.wasm)
+Download the hello.wasm file [here](https://wasmedge.org/docs/files/hello.wasm), or run the following command:
 
 ```bash
-wget https://wasmedge.org/books/app-dev-guide/en/files/hello.wasm
```
Comment (juntao) [inline docs/develop/getting-started/quick_start.md:38]: I think we need to use this url. The documentation and the sample file it refers to should ideally be in the same repo.

File: docs/develop/getting-started/quick_start.md [hunk 5]
```diff
@@ -63,14 +67,14 @@ To learn more about how to create Wasm apps in Rust
...
-## Run an HTTP server
+## How to run an HTTP server
 
 This example is a standalone HTTP server written in Rust. It demonstrates that Rust + WasmEdge as a lightweight stack for microservices. Its source code and build instructions are available [here](https://github.com/second-state/rust-examples/tree/main/server).
 
-[Download the server.wasm file here](/files/server.wasm)
+Download the server.wasm file [here](https://wasmedge.org/docs/files/server.wasm), or run the following command:
 
 ```bash
-wget https://wasmedge.org/books/app-dev-guide/en/files/server.wasm
```
Comment (juntao) [inline docs/develop/getting-started/quick_start.md:73]: See the above comment

File: docs/develop/getting-started/quick_start.md (hunk 4 above)
Comment (kokamkarsahil) [reply docs/develop/getting-started/quick_start.md:38]: Agreed, it makes a lot of sense in doing that way and can avoid tempering files too by building it in actions and storing it artifact. I was wondering which will be a better way for building all examples using Cargo Workspace or building it by triggering each example to build?

I will create a PR in

Comment (Mackenzie-OO7) [reply docs/develop/getting-started/quick_start.md:38]: > I think we need to use this url. The documentation and the sample file it refers to should ideally be in the same repo.

Okay, noted.
Thanks for reviewing!

**Review by alabulei1 — APPROVED** [review]

**Comment by alabulei1** [comment]
Thanks @Mackenzie-OO7, @adithyaakrishna and @juntao

**Pull request merged**